yarn.lock

```diff
@@ -435,7 +435,7 @@
     lodash "^4.2.0"
     to-fast-properties "^2.0.0"
 
-"@textile/go-mobile@^0.1.9":
+"@textile/go-mobile@0.1.9":
```

Comment:
Not sure why this wasn't already updated, but this change happened when I ran `yarn install`

Reply:
After rebasing on latest master